plugins/polio/budget/models.py

```diff
@@ -96,6 +96,23 @@ def __repr__(self):
         return f"{self.step}, {self.alias}"
 
 
+class BudgetProcess(SoftDeletableModel):
+    class Meta:
+        ordering = ["-updated_at"]
+        verbose_name_plural = "Budget Processes"
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    created_by = models.ForeignKey("auth.User", on_delete=models.PROTECT)
+    updated_at = models.DateTimeField(auto_now=True)
+    # Keep trace of the Team the user was acting on behalf of in case they get remove from it.
+    created_by_team = models.ForeignKey("iaso.Team", on_delete=models.PROTECT, blank=True, null=True)
```

Comment:
It's a mistake here. Using it as a string when the model is in a different app is the right way to import a model in Django... It can also avoid circular import. But from django doc, importing as a string when the model is in a different app is the right way to do it.